exist-versioning-release.md

```diff
@@ -150,48 +231,72 @@ You will require a system with:
             <!-- eXist-db Github Release -->
             <server>
                 <id>github</id>
-                <privateKey>[Github Personal access tokens]</privateKey>
+                <privateKey>YOUR-Github-ACCESS-TOKEN</privateKey>
             </server>
         </servers>
+        
```

Comment:
The other servers are missing from this, including:
*  `sonatype-nexus-staging`
*  `exist-db-snapshots`
* `docker.io`

Reply:
Sonatype-nexus-staging and docker.io are left intact and are therefore not shown in this diff.
exist-db-snapshots is not needed for a product release according to you and thus left out.